2.cellprofiler_processing/cputils.py

```diff
@@ -7,28 +7,27 @@
 import pathlib
 from pathlib import Path
```

Comment:
This import may be unnecessary given that you can just use `pathlib.Path`

Reply:
Addressed in previous comment.